feature/networkinstance/networkinstance.go

```diff
@@ -23,13 +23,16 @@ import (
 
 	"github.com/openconfig/featureprofiles/yang/oc"
 	"github.com/openconfig/ygot/ygot"
+	"strconv"
 )
 
 // NetworkInstance struct stores the OC attributes.
 type NetworkInstance struct {
 	oc oc.NetworkInstance
 }
 
+var prefix string
```

Comment:
why is this var global? We should have any global variables.

Reply:
Removed!